### PR TITLE
Code changes for updating correct status in service catalog in case service creation fails

### PR DESCRIPTION
**Pull request by sidddutta**
Service status is updated as failed when Jenkins API call results in an error.

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jazz_create-serverless-service/components/crud/update.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	var svcPayload = {
+		uri: inputs.SERVICE_API_URL + inputs.SERVICE_API_RESOURCE + "/" + service_id,
+		method: 'PUT',
+		headers: {'Authorization': inputs.TOKEN },
+		json: jsonBody,
+		rejectUnauthorized: false
+	};	
+	
+	request(svcPayload, function (error, response, body) {
+		if (response.statusCode === 200 && typeof body !== undefined && typeof body.data !== undefined) {
```
Comment (suryajak) [inline jazz_create-serverless-service/components/crud/update.js]: Can we use the short form `body && body.data`

File: jazz_create-serverless-service/index.js [hunk 2]
```diff
@@ -73,7 +75,26 @@ module.exports.handler = (event, context, cb) => {
             })
             .catch(function (err) {
                 logger.error('Error while creating a service : ' + JSON.stringify(err));
-                return cb(JSON.stringify(errorHandler.throwInternalServerError(err.message)));
+                if(err.jenkins_api_failure) {
+                        logger.error('Jenkins API Failed!!!');
```
Comment (suryajak) [inline jazz_create-serverless-service/index.js]: We already logged the error above (line 77) so this is redundant.

Comment (sidddutta) [reply jazz_create-serverless-service/index.js]: Removed the log.

File: jazz_create-serverless-service/components/crud/update.js (hunk 1 above)
Comment (sidddutta) [reply jazz_create-serverless-service/components/crud/update.js]: Changed as suggested.

**Comment by sidddutta** [comment]
PR Comments are incorporated.

**Pull request merged**